### PR TITLE
refactor: Import react-select from @trezor/components

### LINKED ISSUE #4327: react-select in used directly in suite package
`packages/suite/src/views/wallet/sign-verify/components/SignAddressInput.tsx`

`packages/suite/src/components/suite/WordInput/index.tsx`

is accessing `react-select` directly. In ideal world react-select should be accessed through facade of `@trezor/components`. `react-select` is not dependency of @trezor/suite` after all.

### PR DESCRIPTION
**Pull request by dahaca**
fix #4327

### CODE REVIEW

**Comment by Nodonisko** [comment]
We should not reexport something from package to package in so complicated way. We should use another of possible solutions:
1. Just add `react-select` as dependency to `suite` - reexporting will made it even more complicated and it will be bundled anyway, so there is no reason why this can't be direct dependency of `suite`
2. Create some universal `Select` component in `components` that will be utilizing `react-select` and export just that component (this is probably way that @mroz22 want).

File: packages/components/react-select-lib.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from 'react-select';
```
Comment (Nodonisko) [inline packages/components/react-select-lib.d.ts]: `d.ts` files should no be used for things like this, it should be used only for declaring type interfaces of files or modules (just import/export types). 

https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html

**Review by Nodonisko — CHANGES_REQUESTED** [review]

File: packages/components/react-select-lib.d.ts (hunk 1 above)
Comment (dahaca) [reply packages/components/react-select-lib.d.ts]: Yes, thank you, forgot to change the extension after trying to do another thing. Tests were also failing.

**Comment by dahaca** [comment]
Personally I do not see any problems with including `react-select` as a dependency. Reexporting it from Select is even weirder imo.

**Review by Nodonisko — APPROVED** [review]
From my point of view this is OK, but @mroz22 definitely should check this PR too.

**Review by mroz22 — APPROVED** [review]
I think the original idea was to abstract anything we need into select component inside @trezor/ocmponents so that we can avoid using select from react-select directly. But I guess there are better things to do, so I am fine with this too.

**Pull request merged**